AndroidSDK/src/com/leanplum/LeanplumNotificationChannels.java

```diff
@@ -0,0 +1,578 @@
+   * @param enableLights True if lights enable for this channel.
+   * @param lightColor Light color for notifications posted to this channel, if the device supports
+   * this feature.
+   * @param enableVibration True if vibration enable for this channel.
+   * @param vibrationPattern Vibration pattern for notifications posted to this channel.
+   */
+  static void createNotificationChannel(Context context, String channelId, String
+      channelName, int channelImportance, String channelDescription, String groupId, boolean
+      enableLights, int lightColor, boolean enableVibration, long[] vibrationPattern) {
+    createNotificationChannel(context, channelId, channelName, channelImportance,
+        channelDescription, groupId, enableLights, lightColor, enableVibration,
+        vibrationPattern, 0, false, false, null);
+  }
+
+  /**
+   * Create push notification channel with provided id, name and importance of the channel.
+   * You can call this method also when you need to update the name or description of a channel, at
+   * this case you should use original channel id.
+   *
+   * @param context The application context.
+   * @param channelId The id of the channel.
+   * @param channelName The user-visible name of the channel.
+   * @param channelImportance The importance of the channel. Use value from 0 to 5. 3 is default.
+   * Read more https://developer.android.com/reference/android/app/NotificationManager.html#IMPORTANCE_DEFAULT
+   * Once you create a notification channel, only the system can modify its importance.
+   * @param channelDescription The user-visible description of the channel.
+   * @param groupId The id of push notification channel group.
+   * @param enableLights True if lights enable for this channel.
+   * @param lightColor Light color for notifications posted to this channel, if the device supports
+   * this feature.
+   * @param enableVibration True if vibration enable for this channel.
+   * @param vibrationPattern Vibration pattern for notifications posted to this channel.
+   * @param locksceenVisibility How to be shown on the lockscreen.
+   * @param bypassDnd Whether should notification bypass DND.
+   * @param showBadge Whether should notification show badge.
+   * @param sound Sound URI.
+   */
+  static void createNotificationChannel(Context context, String channelId, String
+      channelName, int channelImportance, String channelDescription, String groupId, boolean
+      enableLights, int lightColor, boolean enableVibration, long[] vibrationPattern, int
+      locksceenVisibility, boolean bypassDnd, boolean showBadge, Uri sound) {
+    if (context == null || TextUtils.isEmpty(channelId)) {
+      return;
+    }
+
+    if (areNotificationChannelsSupported(context)) {
+      try {
+        NotificationManager notificationManager = context.getSystemService(NotificationManager.class);
+        if (notificationManager == null) {
+          Log.e("Notification manager is null");
```

Comment:
If this is customer facing, please use `Log.e("Notification Manager doesn't exist.");`